lib/config.ts

```diff
@@ -104,6 +104,13 @@ export interface Config {
    */
   useBlockingProxy?: boolean;
 
+  /**
+   * If specified, Protractor will connect to the Blocking Proxy at the given
+   * url instead of starting it's own. Intended for debugging Protractor
+   * features only.
```

Comment:
Do we need this 'intended for' warning? There's no big issue with users doing it this way, right?